src/tests/containers/LoginContainer.test.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import configureStore from "redux-mock-store";
+import { shallow, mount } from "enzyme";
+import thunk from "redux-thunk";
+import moxios from "moxios";
+import store from "../../store";
+import LoginContainer, {
+  LoginContainer as DumpLoginContainer,
+  mapStateToProps
+} from "../../containers/Login/LoginContainer";
+
+describe("login container", () => {
+  const initialState = {
+    login: { isSuccesfull: false, token: "", errors: null }
+  };
+  const mockFn = jest.fn();
+  const mockStore = configureStore([thunk]);
+  let store;
+
+  beforeEach(() => {
+    store = mockStore(initialState);
+    moxios.install();
+  });
+
+  afterEach(function() {
+    moxios.uninstall();
+  });
+  it("should render without crashing", () => {
+    const wrapper = shallow(<DumpLoginContainer open={false} close={mockFn} />);
+    expect(wrapper).toMatchSnapshot();
+  });
+  it("should render change", () => {
+    const wrapper = shallow(<DumpLoginContainer open={false} close={mockFn} />);
+    const event = {
+      target: {
+        name: "name"
+      }
+    };
+    wrapper.instance().handleChange(event);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it("renders submit handler", () => {
+    const props = {
+      userLoginRequest: jest.fn()
+    };
+    let wrapper = shallow(<DumpLoginContainer {...props} />);
+    const instance = wrapper.instance();
+    const event = {
+      target: {
+        name: ""
+      }
+    };
+    event.preventDefault = jest.fn();
+    instance.handleSubmit(event);
+  });
+
+  it("handles mapping state to props", () => {
+    const state = {
+      auth_login: { isProcessing: false }
+    };
+    mapStateToProps(state);
```

Comment:
You can't have a test without expecting or asserting something.

Reply:
Done